src/manifests_workflow/input_manifests.py

```diff
@@ -160,3 +171,31 @@ def add_to_cron(self, version: str) -> None:
             f.write(data)
 
         logging.info(f"Wrote {jenkinsfile}")
+
+    def add_to_versionincrement_workflow(self, version: str) -> None:
+        versionincrement_workflow_file = self.versionincrement_workflow()
+        yaml = ruamel.yaml.YAML()
+        yaml.explicit_start = True  # type: ignore
+        yaml.preserve_quotes = True  # type: ignore
+
+        with open(versionincrement_workflow_file) as f:
+            data = yaml.load(f)
+
+        version_entry = []
```

Comment:
This part is not tested.

Reply:
Let me add this as well, I will raise a new PR